susemanager/src/mgr_bootstrap_data.py

```diff
@@ -14,93 +14,7 @@
 # * optional. Example: "a*" if "a" is available add it, otherwise ignore it
 
 
-PKGLIST11 = [
-    "dbus-1-python",
-    "libcurl4",
-    "libnl",
-    "libopenssl0_9_8",
-    "libsqlite3-0",
-    "libxml2-python",
-    "libzypp",
-    "openssl",
-    "python",
-    "python-dmidecode",
-    "python-ethtool",
-    "python-openssl",
-    "python-xml",
-    "rpm-python",
-    "satsolver-tools",
-    "slang",
-    "zypper",
-]
-
-ENHANCE11 = [
-    "libyaml-0-2",
-    "libzmq3",
-    "python-backports.ssl_match_hostname",
-    "python-certifi",
-    "python-curl",
-    "python-futures",
-    "python-Jinja2",
-    "python-MarkupSafe",
-    "python-msgpack-python",
-    "python-psutil",
-    "python-pycrypto",
-    "python-pyzmq",
-    "python-requests",
-    "python-simplejson",
-    "python-tornado",
-    "python-yaml",
-    "salt",
-    "salt-minion",
-]
-
-PKGLIST11_X86_I586 = [
-    "pmtools",
-]
-
-PKGLIST12 = [
-    "libopenssl1_0_0",
-    "openssl",
-    "libzypp",
-    "rpm-python",
-    "libsolv-tools",
-    "zypper",
-    "cron",
-    "cronie",
-]
-
-SLE12VENV = ["logrotate", "libsqlite3-0", "venv-salt-minion"]
-
-ENHANCE12SP1 = ["libyui-ncurses-pkg7", "libyui-qt-pkg7"]
-
-RES6 = [
-    "salt",
-    "salt-minion",
-    "python-futures",
-    "python-jinja2",
-    "python-msgpack-python",
-    "python-psutil",
-    "python-pycrypto",
-    "python-requests",
-    "python-setuptools",
-    "python-tornado",
-    "python-zmq",
-    "zeromq",
-    "openssl",
-    "python-certifi",
-    "PyYAML",
-    "python-markupsafe",
-    "python-urllib3",
-    "libyaml",
-    "python-chardet",
-    "python-six",
-    "yum-plugin-security",
-    "yum",
-    "rpm-python",
-    "redhat-rpm-config",
-    "openssh-clients",
-]
+PKGLIST12 = ["logrotate", "libsqlite3-0", "venv-salt-minion"]
```

Comment:
Can we use a similar  syntax harmonizing  how we write  the list?
If we use something like this, if/when we will have to ad/remove packages, the diff should be less messy from  what I remember.

```suggestion
PKGLIST12 = [
   "logrotate",
   "libsqlite3-0",
   "venv-salt-minion",
]
```

Reply:
The linter revert this.